<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VpnSecurityWidget</class>
 <widget class="QWidget" name="VpnSecurityWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <widget class="QRadioButton" name="typicalRadioButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>392</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Typical (recommended settings)</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
       <attribute name="buttonGroup">
        <string notr="true">buttonGroup</string>
       </attribute>
      </widget>
     </item>
     <item row="1" column="0" rowspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>18</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="passwordCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Require secured password</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QCheckBox" name="encryptionCheckBox">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>Require data encryption (disconnect if none)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QRadioButton" name="advancedRadioButton">
     <property name="text">
      <string>Advanced (custom settings)</string>
     </property>
     <attribute name="buttonGroup">
      <string notr="true">buttonGroup</string>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="dataLabel">
          <property name="text">
           <string>Data encryption:</string>
          </property>
          <property name="buddy">
           <cstring>dataComboBox</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="dataComboBox">
          <item>
           <property name="text">
            <string>None</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Optional</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Required</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Maximum strangth</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QGroupBox" name="logonGroupBox">
        <property name="title">
         <string>Logon security</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <widget class="QRadioButton" name="eapRadioButton">
           <property name="text">
            <string>Use Exstensible Authentication Protocol (EAP)</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="otherRadioButton">
           <property name="text">
            <string>Use these other protocols:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QFrame" name="frame_2">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1" colspan="2">
             <widget class="QCheckBox" name="papCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Uncrypted password (PAP)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>18</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="1" colspan="2">
             <widget class="QCheckBox" name="spapCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Shiva Password Authentication Protocol (SPAP)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1" colspan="2">
             <widget class="QCheckBox" name="chapCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Challenge Handshake Authentication Protocol (CHAP)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1" colspan="2">
             <widget class="QCheckBox" name="mschapCheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Microsoft CHAP (MS-CHAP)</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>36</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="4" column="2">
             <widget class="QCheckBox" name="oldMschapCheckBox">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Older MS-CHAP version for Windows 95 servers</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1" colspan="2">
             <widget class="QCheckBox" name="mschap2CheckBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string>Microsoft CHAP Version 2 (MS-CHAP-v2)</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
        <zorder>frame_2</zorder>
        <zorder>eapRadioButton</zorder>
        <zorder>otherRadioButton</zorder>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="useLogonCheckBox">
     <property name="text">
      <string>Use Windows logon name and password (and domain if any)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
  <zorder>frame</zorder>
  <zorder>advancedRadioButton</zorder>
  <zorder>useLogonCheckBox</zorder>
  <zorder>verticalSpacer</zorder>
 </widget>
 <tabstops>
  <tabstop>typicalRadioButton</tabstop>
  <tabstop>passwordCheckBox</tabstop>
  <tabstop>encryptionCheckBox</tabstop>
  <tabstop>advancedRadioButton</tabstop>
  <tabstop>dataComboBox</tabstop>
  <tabstop>eapRadioButton</tabstop>
  <tabstop>otherRadioButton</tabstop>
  <tabstop>papCheckBox</tabstop>
  <tabstop>spapCheckBox</tabstop>
  <tabstop>chapCheckBox</tabstop>
  <tabstop>mschapCheckBox</tabstop>
  <tabstop>oldMschapCheckBox</tabstop>
  <tabstop>mschap2CheckBox</tabstop>
  <tabstop>useLogonCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>typicalRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>encryptionCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>81</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>typicalRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>passwordCheckBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>170</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>183</x>
     <y>52</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>advancedRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>otherRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>214</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>214</x>
     <y>342</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mschapCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>oldMschapCheckBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>357</y>
    </hint>
    <hint type="destinationlabel">
     <x>236</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
